### DISCUSSION TITLE
Is it possible to have prefix'ed keybindings?

### DISCUSSION THREAD

**Discussion by alex35mil** [zellij-org/zellij#2073] [Q&A]
After using Zellij for some time with configured modes to minimize conflicts with my editor, I still occasionally find myself in the editor in Zellij's normal mode and unexpectedly terminate sessions.

Is there a way (or plans to support) prefix'ed keybindings? `<C-Space> KEY` proved to be the most bulletproof way of preventing conflicts between the terminal multiplexer and editor for me.

**Comment by renannprado** [comment]
I was about to install zellij and I have exactly the same concern: how one can workaround the shortcut conflicts.

Any ideas, good practices, etc would be helpful.

**Comment by alex35mil** [comment]
@renannprado When I ran Nvim inside Zellij, I used [this setup](https://www.reddit.com/r/vimporn/comments/10eqj6z/neovim_zellij/j4wsfbh/). It worked for me.

**Reply by renannprado** [reply]
Thanks! I'll have a look. 

Is there anything specific to `zsh` conflicts as well? e.g. I'm not zellij user yet, but I'm talking about `CTRL+R`, for example.

**Reply by Lockszmith-GH** [reply]
`[Ctrl]`+`[R]` works without issues, the defaults are very sensible.
The default key to block all keys is `[Ctrl]`+`[G]`, and it can easily be configured to something else.

BTW, the docs for zellij are good, and easily searchable:  
https://zellij.dev/documentation/faq.html?highlight=ctrl-g#zellij-overrides-certain-key-combinations-that-i-use-for-other-apps-what-can-i-do
